test/test_macho.rb

```diff
@@ -46,6 +46,16 @@ def test_load_commands
     end
   end
 
+  def test_load_commands_well_ordered
+    filenames = SINGLE_ARCHES.map { |a| fixture(a, "libhello.dylib") }
+
+    filenames.each do |fn|
+      file = MachO::MachOFile.new(fn)
+      offsets = file.load_commands.map(&:view).map(&:offset)
```

Comment:
I don't have very strong feelings regarding this (thus feel free to ignore and keep as-is), but I'm leaning towards the following instead of chaining multiple `map` calls:

```ruby
offsets = file.load_commands.map { |lc| lc.view.offset }
```